cmd/fyne/internal/util/file.go

```diff
@@ -39,9 +39,8 @@ func EnsureSubDir(parent, name string) string {
 	return path
 }
 
-func copyFileMode(src, tgt string, perm os.FileMode) error {
-	_, err := os.Stat(src)
-	if err != nil {
+func copyFileMode(src, tgt string, perm os.FileMode) (err error) {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
That does not matter, no.

And definitely not use a `naked return`.

Reply:
There even is a proposal to entirely remove `naked returns` because of the extra overhead it has for the reader of the code:  https://github.com/golang/go/issues/21291

Reply:
Oh got it, that's a good point.